src/main/resources/com/ncc/aif/restricted_hypothesis_aif.shacl

```diff
@@ -25,6 +25,59 @@ aida:SystemShape
         """ ;
     ] .
 
+#########################.;
+# 4.3 TA3 #4 Each hypothesis graph must have exactly one hypothesis importance value
+#------------------------
+aida:HypothesisImportanceRequiredShape
+    a sh:PropertyShape ;
+    sh:path aida:importance ;
+    sh:or (
```

Comment:
Why include this?

Reply:
Great point. I originally did not have this in here but I had issues with the tests validating and I think this was a byproduct of trying to get everything to work and ultimately I didn't even need it. I'll remove and update.